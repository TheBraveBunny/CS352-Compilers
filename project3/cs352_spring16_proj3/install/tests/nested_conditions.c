int thing;

int foo(int i){
   while(i) {
	   int a;
	   int b;
	   if(i> 0){
	       a = 1;
	       b = a;
	       printf("in if\n");
	   }
	   else{
		a=2;
		int c;
	   }
   	
	a=b;
   	printf("a = %d", a);
   }
   return 1;
}
main(){
   foo(0);
}

