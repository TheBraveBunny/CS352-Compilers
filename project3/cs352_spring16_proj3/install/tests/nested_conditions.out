#functions: 2
#global vars: 1
#local vars: 3
#statements: 13
